ui/cypress/e2e/dashboardsView.test.ts

```diff
@@ -21,7 +21,7 @@ describe('Dashboard', () => {
     })
   })
 
-  it('can edit a dashboards name', () => {
+  it("can edit a dashboard's name", () => {
```

Comment:
double quotes on purpose?

Reply:
it used to be

>can edit a dashboards name

so i added the possessive apostrophe and the double quote so we wouldn't have to escape the apostrophe